---
int8 num_objects
bool in_goal
ReconfigurationObject[] objects